examples/cms-sanity/README.md

```diff
@@ -70,7 +70,7 @@ If you started with [deploying your own](#deploy-your-own) then you can run this
 
 ```bash
 npx vercel link
-npx vercel pull
+npx vercel env pull
```

Comment:
```suggestion
npx vercel pull
```

To reuse remote vars, `vercel pull` is correct.
x-ref: https://vercel.com/docs/cli/pull#vercel-pull

Reply:
We need the files to be pulled into a `.env` file, otherwise `next dev` won't see them:
![image](https://github.com/user-attachments/assets/9150dfd1-3fb0-4a11-8c70-193ee2af3f4b)

Reply:
You copy from the `.env.example` here: https://github.com/vercel/next.js/blob/4cdc12a633f234efa0fe7b1dd4eb98bae2c22ba0/examples/cms-sanity/README.md#using-the-sanity-cli

You won't have any envs to pull since the project is just initiated.

Reply:
Ah, you're right, sorry about that.